galsim/phase_psf.py

```diff
@@ -1385,7 +1394,7 @@ def __init__(self, lam_over_diam=None, lam=None, diam=None, tip=0., tilt=0., def
         optics_screen = galsim.OpticalScreen(
                 defocus=defocus, astig1=astig1, astig2=astig2, coma1=coma1, coma2=coma2,
                 trefoil1=trefoil1, trefoil2=trefoil2, spher=spher, aberrations=aberrations,
-                lam_0=lam)
+                obscuration=obscuration, annular_zernike=annular_zernike, lam_0=lam)
```

Comment:
Not a bug fix.  When the Zernikes are circular, the obscuration isn't needed to set up the `OpticalScreen`.  Now that we support annular Zernikes, though, we need to specify the annulus.

I went ahead and added `annular_zernikes` and `obscuration` to `__str__`, since that's still pretty light weight.